core/src/functions/moveElementInArray/moveElementInArray.ts

```diff
@@ -0,0 +1,16 @@
+export const moveElementInArray = <T>(
```

Comment:
tsdocs would be nice :) Just a quick explanation what the function does and explaining what happens when using too large or too small indexes etc.

Reply:
Done.